assets/multiqc_config.yml

```diff
@@ -39,9 +34,33 @@ module_order:
       name: "GATK4 MarkDuplicates"
       info: " metrics generated either by GATK4 MarkDuplicates or EstimateLibraryComplexity (with --use_gatk_spark)."
   - samtools:
-      name: "Samtools Flagstat"
+      name: "Samtools Flagstat (mapped)"
+      fn: "*.md.cram.stats"
+      anchor: "md_stats"
+      info: "This section of the report shows mapped samtools stats."
+      path_filters:
+        - "*.md.cram.stats"
+  - mosdepth:
+      name: "Mosdepth (mapped)"
+      fn: "*.mapped.mosdepth.*.dist.txt"
+      anchor: "mapped_mosdepth"
+      info: "This section of the report shows the mapped distribution after mosdepth."
+      path_filters:
+        - "*.mapped.mosdepth.*.dist.txt"
+  - samtools:
```

Comment:
so is the solution to have two plots instead of one? (Fine for me, just curious 😊 )

Reply:
No it's still not working but that is not a fault of the multiqc report, that's rather because the files do not get published to the multiqc work directory. I can remove that section and work on samtools/mosdepth in a new PR if you prefer that. Then we could already incorporate the bugfixes and this PR is already so large.